Unicode/makeutype.c

```diff
@@ -128,11 +129,20 @@ unsigned long flags2[MAXC];
 unichar_t alts[MAXC][MAXA+1];
 unsigned long assignedcodepoints[0x120000/32];	/* 32 characters represented per each long value */
 
+int frc, frm, lgc, lgm, vfc, vfm;		/* identify all ligatures and fractions */
+#define LG_MAX 550
+#define FR_MAX 100
+#define VF_MAX 50
+unsigned long ligature[LG_MAX];
+unsigned long fraction[FR_MAX];
+unsigned long vulgfrac[VF_MAX];
```

Comment:
Gah. More global variables. I suppose there's no sense in sessionizing just some of them, though.